PayloadPkg/FirmwareUpdate/FirmwareUpdate.c

```diff
@@ -900,7 +946,7 @@ ApplyFwImage (
       CsmeUpdateInData = InitCsmeUpdInputData();
       if (CsmeUpdateInData != NULL) {
         Status = UpdateCsme(CapImage, CapImageSize, CsmeUpdateInData, ImageHdr);
-        *ResetRequired = TRUE;
+        *ResetRequired = FALSE;

```

Comment:
Are there usages where reset may be required after CSME update ?

Reply:
Here is flow with various Fw capsule updates,
Only CSME component with Capsule - Anyways reset would happen by FwUpdate payload to start a regular boot.
CSME + SBL BIOS  in capsule - First CSME will be updated. Next BIOS B partition, reboot. Next update BIOS A partitions & reboot.
CSME + SBL BIOS + Container in capsule - container would be taken in last iteration.

In either of cases reboot happen after CSME update before next regular boot.